tools/ramclustr/ramclustr.xml

```diff
@@ -0,0 +1,62 @@
+<tool id="ramclustr" name="RAMClustR" version="@TOOL_VERSION@+galaxy1">
+    <macros>
+        <import>ramclustr_macros.xml</import>
+    </macros>
+
+    <requirements>
+        <container type="docker">recetox/ramclustr:1.1.0-recetox0</container>
+    </requirements>
+
+    <inputs>
+        <param name="input_xcms" label="input_xcms" type="data" format="RData" help=": containing grouped feature data for clustering by ramclustR" />
+        <expand macro="parameters" />
+    </inputs>
+
+    <command detect_errors="aggressive"><![CDATA[
+    
+        Rscript 
+            -e 'obj <- load("$input_xcms")'
+            -e 'x <- RAMClustR::ramclustR(
+                xcmsObj = xdata,
+                #if $params.MStag 
+                    MStag = "$params.MStag",
+                #end if
+                #if $params.idMSMStag 
+                    idMSMStag = "$params.idMSMStag",
+                #end if
+                taglocation = "$params.taglocation",
+                sr = $params.sr,
+                #if $params.st 
+                    st = $params.st,
+                #end if
+                #if $params.maxt 
+                    maxt = $params.maxt,
+                #end if
+                deepSplit = $params.deepSplit,
+                blocksize = $params.blocksize,
+                mult = $params.mult,
+                hmax = $params.hmax,
+                collapse = $params.collapse,
+                usePheno = $params.usePheno,
+                mspout = $params.mspout,
+                qc.inj.range = $params.qc_inj_range,
+                normalize = "$params.normalize",
+                minModuleSize = $params.minModuleSize,
+                linkage = $params.linkage,
+                mzdec = $params.mzdec,
+                cor.method = "$params.cor_method",
+                rt.only.low.n = $params.rt_only_low_n,
+                replace.zeros = $params.replace_zeros
+                )'
+            -e 'save(x, file = "$result")'
```

Comment:
Same comment as on the line 18.

Reply:
The output of the tool is still handled preliminary.